package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1985,6 +1995,13 @@ def align_principal_axis(group, axis, vector):
     transplants[GroupBase].append(
         ('align_principal_axis', align_principal_axis))
 
+    @classmethod
+    def is_value_missing(self, value):
+        """check if an attribute has a missing value
```

Comment:
```suggestion
        """Check if an attribute has a missing value
        
```
I think you may need a newline here to get the documentation to render properly.

<img width="664" alt="Screenshot 2023-12-20 at 6 54 57 pm" src="https://github.com/MDAnalysis/mdanalysis/assets/31115101/fc4766fd-6dda-4719-a07e-e5ff90737ad1">